distribution/test/itests/test-itests-alliance/src/test/java/org/codice/alliance/test/itests/ImagingTest.java

```diff
@@ -155,11 +144,278 @@ public void testImageNitfChipCreation() throws Exception {
+        }
+
+        private MetacardXmlValidator has(String type, String name, String expectedValue) {
+            response.body(hasXPath(String.format("/metacard[@id='%s']/%s[@name='%s']/value",
+                    id,
+                    type,
+                    name), is(expectedValue)));
+            return this;
+        }
+
+        /**
+         * Chains two conditions
+         *
+         * @return this validator
+         */
+        public MetacardXmlValidator and() {
+            return this;
+        }
+
+        /**
+         * Asserts that the response contains a {@code <string>} element with a specific value
+         *
+         * @param name          string element name as specified by the {@code name} attribute
+         * @param expectedValue text expected in the {@code <value>} element
+         * @return this validator
+         */
+        public MetacardXmlValidator hasStringElement(String name, String expectedValue) {
+            return has("string", name, expectedValue);
+        }
+
+        /**
+         * Asserts that the response contains an {@code <int>} element with a specific value
+         *
+         * @param name          int element name as specified by the {@code name} attribute
+         * @param expectedValue integer expected in the {@code <value>} element
+         * @return this validator
+         */
+        public MetacardXmlValidator hasIntElement(String name, Integer expectedValue) {
+            return has("int", name, expectedValue.toString());
+        }
+
+        /**
+         * Asserts that the response contains a {@code <dateTime>} element with a specific value
+         *
+         * @param name          dateTime element name as specified by the {@code name} attribute
+         * @param expectedValue date and time expected in the {@code <value>} element
+         * @return this validator
+         */
+        public MetacardXmlValidator hasDateElement(String name, String expectedValue) {
+            return has("dateTime", name, expectedValue);
+        }
+
+        /**
+         * Asserts that the response contains a {@code <base64Binary>} element with a specific
+         * value
+         *
+         * @param name     base64Binary element name as specified by the {@code name} attribute
+         * @param fileName name of the file that contains the expected base 64 encoded text in
+         *                 the {@code <value>} element
+         * @return this validator
+         */
+        public MetacardXmlValidator hasBase64Binary(String name, String fileName)
+                throws IOException {
+            InputStream inputStream = getClass().getClassLoader()
+                    .getResourceAsStream(fileName);
+            String expectedValue = IOUtils.toString(inputStream);
+            return has("base64Binary", name, expectedValue);
+        }
+    }
+
+    private MetacardXmlValidator ingestAndValidateCommonNitfJpeg2000Attributes(int imageNumber)
```

Comment:
I think this image index lookup is a bit fragile. Could you change from an array of the files to a lookup map?